pkgs/tools/misc/bkt/default.nix

```diff
@@ -0,0 +1,30 @@
+{ rustPlatform
+, fetchFromGitHub
+, lib
+, doCheck ? true
+, withFeatures ? p: p
+}: rustPlatform.buildRustPackage rec {
+  inherit doCheck;
+
+  pname = "bkt";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "dimo414";
+    repo = pname;
+    rev = "4b2e3b47e9ca4fdcdc82a01e5ec733a389f81e13";
```

Comment:
inb4: this is necessary because the version change in the `Cargo.lock` is not included in the release for `0.5.4`, the diff between this commit and `0.5.4` is exactly that update. See https://github.com/dimo414/bkt/commit/4b2e3b47e9ca4fdcdc82a01e5ec733a389f81e13.

Reply:
0.6.0 is now out which should not have this issue.